extensions/hibernate-search-orm-coordination-outbox-polling/runtime/src/main/java/io/quarkus/hibernate/search/orm/coordination/outboxpolling/runtime/HibernateSearchOutboxPollingBuildTimeConfigPersistenceUnit.java

```diff
@@ -0,0 +1,167 @@
+
+        /**
+         * The name of the agent table.
+         *
+         * @asciidoclet
+         */
+        @ConfigDocDefault("HSEARCH_AGENT")
+        Optional<String> table();
+
+        /**
+         * The UUID generator strategy used for the agent table.
+         *
+         * Available strategies:
+         *
+         * * `auto` (the default) is the same as `random` which uses `UUID#randomUUID()`.
+         * * `time` is an IP based strategy consistent with IETF RFC 4122.
+         *
+         * @asciidoclet
+         */
+        @ConfigDocDefault("auto")
+        Optional<UuidGenerationStrategy> uuidGenStrategy();
+
+        /**
+         * The name of the Hibernate ORM basic type used for representing an UUID in the agent table.
+         *
+         * Refer to
+         * https://docs.jboss.org/hibernate/orm/6.2/userguide/html_single/Hibernate_User_Guide.html#basic-legacy-provided[this
+         * section of the Hibernate ORM documentation]
+         * to see the list of available UUID representations provided by Hibernate ORM.
+         *
+         * A user defined type can also be supplied.
+         *
+         * Defaults to the special value `default`, which will result into one of `uuid`/`uuid-binary`/`uuid-char`
+         * depending on the database kind.
+         *
+         * @asciidoclet
+         */
+        @ConfigDocDefault("uuid/uuid-binary/uuid-char depending on the database kind")
+        Optional<String> uuidType();
+
+    }
+
+    @ConfigGroup
+    interface EntityMappingOutboxEventConfig {
+
+        /**
+         * The database catalog to use for the outbox event table.
+         *
+         * @asciidoclet
+         */
+        @ConfigDocDefault("Default catalog configured in Hibernate ORM")
+        Optional<String> catalog();
+
+        /**
+         * The schema catalog to use for the outbox event table.
+         *
+         * @asciidoclet
+         */
+        @ConfigDocDefault("Default schema configured in Hibernate ORM")
+        Optional<String> schema();
+
+        /**
+         * The name of the outbox event table.
+         *
+         * @asciidoclet
+         */
+        @ConfigDocDefault("HSEARCH_OUTBOX_EVENT")
+        Optional<String> table();
+
+        /**
+         * The UUID generator strategy used for the outbox event table.
+         *
+         * Available strategies:
+         *
+         * * `auto` (the default) is the same as `random` which uses `UUID#randomUUID()`.
+         * * `time` is an IP based strategy consistent with IETF RFC 4122.
+         *
+         * @asciidoclet
+         */
+        @ConfigDocDefault("auto")
+        Optional<UuidGenerationStrategy> uuidGenStrategy();
+
+        /**
+         * The name of the Hibernate ORM basic type used for representing an UUID in the outbox event table.
+         *
+         * Refer to
+         * https://docs.jboss.org/hibernate/orm/6.2/userguide/html_single/Hibernate_User_Guide.html#basic-legacy-provided[this
```

Comment:
do you know if this doclet supports any properties so we can pass URLs to javadocs?

Reply:
I think it's possible,  I just didn't bother. I'll add a commit.